ios/BetterRailWidget/Live Activity/BetterRailLiveActivity.swift

```diff
@@ -23,19 +23,18 @@ struct BetterRailLiveActivity: Widget {
       return DynamicIsland {
             // Expanded UI goes here.  Compose the expanded UI through
             // various regions, like leading/trailing/center/bottom
-          DynamicIslandExpandedRegion(.leading, priority: 90.0) {
+          DynamicIslandExpandedRegion(.leading) {
             LeadingView(vm: vm)
               .dynamicIsland(verticalPlacement: .belowIfTooWide)
+              .padding(.leading, 4)
           }
           DynamicIslandExpandedRegion(.trailing) {
-              VStack {
-                Spacer()
-                TimeInformation(vm: vm, placement: .dynamicIsland)
-              }
-            }
-            DynamicIslandExpandedRegion(.bottom) {
-              RideInformationBar(vm: vm, placement: .dynamicIsland)
-            }
+            VStack {
+              Spacer()
+              TimeInformation(vm: vm, placement: .dynamicIsland)
+            }.padding(.trailing, 4)
+          }
+          DynamicIslandExpandedRegion(.bottom) {}
```

Comment:
Why did you remove the ride bar? It has valuable information, even in the dynamic island.

Reply:
restored:
<img width="332" alt="Screenshot 2023-05-20 at 4 22 40" src="https://github.com/better-rail/app/assets/13344923/4dd4c9f8-05fc-4446-b5f8-1344f737c5ac">